### ISSUE TITLE
coordinate resolution, symbol vs pixel rendering

### ISSUE THREAD

**Issue by ramagottfried** [ramagottfried/symbolist#82]
it would be nice if symbols could have sub-pixel accuracy, unfortunately many component positioning tools in JUCE are in terms of int, so we may need to have separate values for "real" positions and "rendered" pixel positions.

**Comment by j-bresson** [comment]
how about just storing positions/sizes as floats ?

**Comment by ramagottfried** [comment]
right, yeah, that's what we're doing already ... I'm dealing with some annoying rounding issues with the scaling preview component, so maybe it will all be clearer/easier after I get that figured out.

**Comment by j-bresson** [comment]
I've noticed that, too.. also for instance (maybe related?) if you scale from the left or top border it makes some strange flickering effects..

**Comment by ramagottfried** [comment]
right! yes, probably related -- I'll keep working on it (but probably should pause now for today).
